recipes/beagle/meta.yaml

```diff
@@ -27,8 +29,9 @@ about:
   home: http://faculty.washington.edu/browning/beagle/beagle.html
   license: GPLv3
   license_family: GPL
```

Comment:
```suggestion
  license: GPL-3.0-or-later
  license_family: GPL3
```

Reply:
Please use a SPDX identifier for the `license` field, otherwise looks good to me!

Reply:
@mencian Thank you for the comment, it's added and tests passed.